src/Activators/FileActivator.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Nwidart\Modules\Activators;
+
+use Illuminate\Cache\CacheManager;
+use Illuminate\Config\Repository as Config;
+use Illuminate\Container\Container;
+use Illuminate\Filesystem\Filesystem;
+use Nwidart\Modules\Contracts\ActivatorInterface;
+use Nwidart\Modules\Module;
+
+class FileActivator implements ActivatorInterface
+{
+	/**
+	 * Laravel cache instance
+	 * 
+	 * @var CacheManager
+	 */
+	protected $cache;
+
+	/**
+	 * Laravel Filesystem instance
+	 * 
+	 * @var Filesystem
+	 */
+	protected $files;
+
+	/**
+	 * Laravel config instance
+	 * @var Config
+	 */
+	protected $config;
+
+	/**
+	 * @var string
+	 */
+	protected $cacheKey;
+
+	/**
+	 * @var string
+	 */
+	protected $cacheLifetime;
+
+	/**
+	 * Array of modules activation statuses
+	 * 
+	 * @var array
+	 */
+	protected $installed;
+
+	/**
+	 * File used to store activation statuses
+	 * 
+	 * @var string
+	 */
+	protected $fileInstalled;
+
+	public function __construct(Container $app)
+	{
+		$this->cache = $app['cache'];
+		$this->files = $app['files'];
+		$this->config = $app['config'];
+		$this->fileInstalled = $this->config('file');
+		$this->cacheKey = $this->config('cache-key');
+		$this->cacheLifetime = $this->config('cache-lifetime');
+		$this->installed = $this->getCached();
+	}
+
+	/**
+	 * Get installed cache
```

Comment:
This comment looks a bit unclear to me. From the comment alone, I expected this to return laravel's cache implementation.